Item/Item.js

```diff
@@ -10,37 +10,92 @@
  * @exports ItemDecorator
  */
 
+import EnactPropTypes from '@enact/core/internal/prop-types';
 import kind from '@enact/core/kind';
 import Spottable from '@enact/spotlight/Spottable';
-import ForwardRef from '@enact/ui/ForwardRef';
-import Slottable from '@enact/ui/Slottable';
+import Pure from '@enact/ui/internal/Pure';
 import {ItemBase as UiItemBase, ItemDecorator as UiItemDecorator} from '@enact/ui/Item';
 import {Cell, Layout, Row} from '@enact/ui/Layout';
-import {Marquee, MarqueeController} from '@enact/ui/Marquee';
-import Pure from '@enact/ui/internal/Pure';
-import compose from 'ramda/src/compose';
+import Slottable from '@enact/ui/Slottable';
 import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
 import React from 'react';
 
-import componentCss from './Item.module.less';
+import {Marquee, MarqueeController} from '../Marquee';
 import Skinnable from '../Skinnable';
 
+import componentCss from './Item.module.less';
+
+/**
+ * Renders the Item content.
+ *
+ * @class ItemContent
+ * @memberof agate/Item
+ * @ui
+ * @public
+ */
 const ItemContent = kind({
 	name: 'ItemContent',
-	propTypes: {
+
+	propTypes:  /** @lends agate/Item.ItemContent.prototype */ {
+		/**
+		 * Text displayed when passed as children.
+		 *
+		 * @type {String | Number}
+		 * @public
+		 */
 		content: PropTypes.any,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * `itemContent` - The root class name
+		 * `content` - The content class name
```

Comment:
I don't see these in `publicClassNames`. Are you going to make it public as well?

Reply:
No, I won't. I thought app developers should know which classNames contains `ItemContent`. I will remove these two and only leave `label`. Do you think this is right?

Reply:
Yeap. We only list the classes which are listed in `publicClassNames` here.